ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -3044,6 +3086,7 @@ static int zend_jit_align_func(dasm_State **Dst)
 	track_last_valid_opline = 0;
 	jit_return_label = -1;
 	|.align 16
+	|	ENDBR
```

Comment:
It's better to move this from ``zend_jit_align_func()`` to ``zend_jit_prologue()``

Reply:
Done.